kale/loaddata/dataset_access.py

```diff
@@ -28,21 +28,49 @@ def get_train(self):
         """
         raise NotImplementedError()
 
-    def get_train_val(self, val_ratio):
+    def get_class_subsampled_train(self, class_ids=None):
+        """
+        Args:
+            class_ids (list, optional): List of class ids that are to be subsampled.
+        Returns: a torch.utils.data.Dataset
+            Dataset: a torch.utils.data.Dataset with only classes in class_ids
+        """
+        train_dataset = self.get_train()
+        if class_ids is None:
+            return train_dataset
+        else:
+            sub_indices = [i for i in range(0, len(train_dataset)) if train_dataset[i][1] in class_ids]
+            return torch.utils.data.Subset(train_dataset, sub_indices)
```

Comment:
See here torch also use subset rather than subsample, try to follow torch naming conventions as much as possible.